src/sentry/stacktraces/processing.py

```diff
@@ -543,6 +573,14 @@ def process_stacktraces(data, make_processors=None, set_raw_stacktrace=True):
                     stacktrace_info.stacktrace["frames"] = new_frames
                     changed = True
                     span.set_data("data_changed", True)
+
+                mechanism = stacktrace_info.container.get("mechanism") if is_exception else None
+                error_type = stacktrace_info.container.get("type") if is_exception else None
+                with sentry_sdk.configure_scope() as scope:
+                    scope.set_tag(
```

Comment:
Is there a way to flag-protect this to only make it available internally for now?

Reply:
Added the flag `organizations:filter-empty-console-errors` for this.